saleae/saleae.py

```diff
@@ -151,6 +152,16 @@ def kill_logic(kill_all=False):
 		for candidate in candidates:
 			candidate.terminate()
 
+	@staticmethod
+	def is_logic_running():
```

Comment:
Could you add a short doc comment for this function here so this gets picked up by the automated tooling?